Prox/Prox/Utilities/EventManager.swift

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import CoreLocation
+
+class EventManager {
+
+    fileprivate var eventFetchCompletionHandler: (([Event]?, Error?) -> Void)?
+    fileprivate var shouldFetchEvents: Bool {
+        guard var lastLocationFetchTime = timeOfLastLocationUpdate else {
+            return false
+        }
+        let now = Date()
+        lastLocationFetchTime += AppConstants.minimumTimeAtLocationBeforeFetchingEvents
+        return lastLocationFetchTime < now
+    }
+
+    fileprivate var timeOfLastLocationUpdate: Date? {
+        return UserDefaults.standard.value(forKey: AppConstants.timeOfLastLocationUpdateKey) as? Date
```

Comment:
There's no longer a setter here and, as per the previous PR, I don't think it's ever set.